modin/pandas/series.py

```diff
@@ -1242,27 +1242,19 @@ def nlargest(self, n=5, keep="first"):  # noqa: PR01, RT01, D200
         """
         Return the largest `n` elements.
         """
-        if len(self._query_compiler.columns) == 0:
-            raise NotImplementedError(
-                "Series.nlargest is not implemented for empty Series."
-            )
-        return DataFrame(
+        return Series(
             query_compiler=self._query_compiler.nlargest(
-                n, [self._query_compiler.columns[0]], keep
+                n=n, columns=self.name, keep=keep
```

Comment:
qq: can you just explain the rationale behind these changes? I assume we had the empty Series check for Modin because of some bug. Will this be resolved in the QC now?

Reply:
Oops, I accidentally removed that for some testing stuff. We still need the empty Series check b/c there is some weird behavior on the Modin side :) -- I don't believe pushdown has any problems however.